lib/Models/CsvCatalogItem.ts

```diff
@@ -65,6 +65,18 @@ export default class CsvCatalogItem extends TableMixin(
     return isDefined(this._csvFile);
   }
 
+  // Zooming to tables with lat/lon columns works
+  get canZoomTo() {
```

Comment:
This should probably be a computed. That will avoid any unnecessary React redraws of the workbench UI/other parts of the UI that use `canZoomTo`.